frontend/VolunteerOne/screens/Friends/notificationsPage.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { StyleSheet, Dimensions, ScrollView } from 'react-native';
+import { Block, theme } from 'galio-framework';
+import { Button } from "../../components";
+
+// import { Card } from '../../components';
+import NotificationItem from '../../components/NotificationItem';
+// import articles from '../../constants/articles';
+import notifications from '../../constants/notifications';
+
+// <matt>
+// import { Block, Button, Text, theme } from "galio-framework";
+// </matt>
+
+const { width } = Dimensions.get('screen');
+
+class Home extends React.Component {
+  renderNotifications = () => {
+    return (
+      <ScrollView
+        showsVerticalScrollIndicator={false}
+        contentContainerStyle={styles.notifications}>
+ 
+        <Block flex>
+          <NotificationItem item={notifications[0]} horizontal  />
+          <NotificationItem item={notifications[1]} horizontal />
+          <NotificationItem item={notifications[2]} horizontal />
+          <NotificationItem item={notifications[3]} horizontal />
+          <NotificationItem item={notifications[4]} horizontal />
```

Comment:
Looks good, but we want to keep dynamic rendering in mind. You can try the js 'map' function which is similar to a for loop and render a <NotificationItem> for each object in constant file.

Reply:
committed